src/main/java/net/minecraftforge/event/entity/ProjectileImpactEvent.java

```diff
@@ -125,4 +126,24 @@ public FireworkRocketEntity getFireworkRocket()
             return fireworkRocket;
         }
     }
+
+    /**
+     * Event is cancellable, causes bobber to ignore the current hit and continue on its journey.
+     */
+    @Cancelable
+    public static class FishingBobber extends ProjectileImpactEvent//TODO figure out what happens on cancel
```

Comment:
Why is this `TODO` still here?

Reply:
because I was apparently blind. fixed now.